Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nodesource/node:4.2.5
+
+RUN apt-get update
+RUN apt-get install -y gnupg libkrb5-dev
+RUN npm install -g gulp-cli
+
+
+COPY . /shellshare/
+
+WORKDIR /shellshare
+RUN npm set unsafe-perm true
+RUN npm install
```

Comment:
It would also be neat to have a healthcheck. I think this should work:

```
HEALTHCHECK --interval=5m --timeout=3s CMD curl -f http://localhost:3000/ || exit 1
```

Reply:
`npm start` is run by the "parent" [container](https://github.com/nodesource/docker-node/blob/master/debian/sid/node/4.2.5/Dockerfile).

:+1: the `HEALTHCHECK` I didn't know about that! Thanks!